### PR TITLE
Introduce T4 to GPU canary test

### PR DESCRIPTION
**Pull request by ameukam**
Reuse existing canary test for GPU scheduling to investigate flakiness on Nvidia T4 GPUs.

### CODE REVIEW

File: config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml [hunk 1]
```diff
@@ -86,7 +93,6 @@ periodics:
       - --gcp-node-image=gci
       - --gcp-nodes=2
       - --gcp-project=k8s-infra-e2e-gpu-project
-      - --gcp-zone=us-west1-b
```
Comment (dims) [inline config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml:89]: we don't need the zone?

Comment (ameukam) [reply config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml:89]: No. Boskos will default to `us-central1`. This canary will validate this assumption.

**Pull request merged**